strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -27,198 +27,112 @@
 class GaussianBackend(BaseGaussian):
     """Gaussian backend implementation"""
     def __init__(self):
-        """
-        Instantiate a GaussianBackend object.
+        """Initialize the backend.
         """
         super().__init__()
         self._supported["mixed_states"] = True
         self._short_name = "gaussian"
+        self._init_modes = None
+        self.circuit = None
 
     def begin_circuit(self, num_subsystems, *, cutoff_dim=None, pure=None, **kwargs):
-        r"""
-        Create a quantum circuit (initialized in vacuum state) with number of modes
-        equal to num_subsystems.
+        r"""Instantiate a quantum circuit.
+
+        Instantiates a representation of a quantum optical state with num_subsystems modes.
+        The state is initialized to vacuum.
```

Comment:
```suggestion
        The state is initialiazed to vacuum.
```

Reply:
I don't think this is right? 'initialized' should be correct

Reply:
lol you're right! Please ignore